apps/src/applab/applab.js

```diff
@@ -1491,6 +1513,7 @@ Applab.execute = function() {
       'if (obj) { var ret = obj.fn.apply(null, obj.arguments ? obj.arguments : null);' +
                  'setCallbackRetVal(ret); }}';
     var session = studioApp.editor.aceEditor.getSession();
+    annotationList.attachToSession(session);
```

Comment:
What's the difference between this `attachToSession` call and the one in the `mode-javascript_codeorg`? Would they end up being the same thing?